robottelo/host_helpers/satellite_mixins.py

```diff
@@ -41,6 +41,10 @@ def enable_puppet_satellite(self):
 
     def enable_multicv_setting(self):
         """Makes multi-CV setting available in the downstream Satellite"""
+        if len(
+            self.api.Setting().search(query={'search': f'name={"allow_multiple_content_views"}'})
```

Comment:
```suggestion
            self.api.Setting().search(query={'search': 'name="allow_multiple_content_views"'})
```
You should be able to pass the search string without brackets / fstring, since there is no variable to pass, non-blocking.

Reply:
Perhaps pre-commit gave you an issue with this? Sometimes i encounter that too with these strings

Reply:
I think this was just a copy-pasta. updated :+1: